infrasim/ipmi.py

```diff
@@ -23,20 +23,29 @@ def status_ipmi():
         print "Infrasim IPMI service is stopped"
 
 
-def start_ipmi(conf=VM_DEFAULT_CONFIG):
+def start_ipmi(conf_file=VM_DEFAULT_CONFIG):
```

Comment:
Module ipmi and module socat are only used in test, for user to start ipmi_sim and socat service respectively.
Module qemu is used to start qemu alone but what's different is qemu module is exactly the entrance called by startcmd.

***Both module can be moved to test.***